### PR TITLE
Fix lint errors in festival-e2e

### PR DESCRIPTION
**Pull request by xmano**
Fix lint errors in festival-e2e.

### CODE REVIEW

File: apps/festival/festival-e2e/src/integration/dashboard/movie-tunnel.spec.ts [hunk 1]
```diff
@@ -228,6 +228,7 @@ const debugMovieId = '';
  * @param id : movie doc ID
  * @param loc : tunnel path (main / summary / etc) to debug.
  */
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const debugMovieTitle = (id: string, loc: string) => {
```
Comment (phpgeekfr) [inline apps/festival/festival-e2e/src/integration/dashboard/movie-tunnel.spec.ts:232]: comment the entire function or remove it entirely

File: apps/festival/festival-e2e/src/integration/dashboard/movie-tunnel.spec.ts [hunk 2]
```diff
@@ -255,7 +256,7 @@ describe('User can navigate to the movie tunnel pages start and main.', () => {
     cy.url().then(url => {
       cy.log(`Adding new movie url: ${url}`);
       movieURL = url;
-      console.log("movie :", url);
+      console.log("movie :", movieURL);
```
Comment (phpgeekfr) [inline apps/festival/festival-e2e/src/integration/dashboard/movie-tunnel.spec.ts:259]: use directly `url` and remove `let movieURL: string;`

File: apps/festival/festival-e2e/src/support/pages/dashboard/EventPage.ts [hunk 3]
```diff
@@ -8,7 +8,7 @@ export default class EventPage {
   }
 
   waitNewUrl() {
-    return new Cypress.Promise((resolve, reject) => {
+    return new Cypress.Promise((resolve) => {
```
Comment (phpgeekfr) [inline apps/festival/festival-e2e/src/support/pages/dashboard/EventPage.ts:11]: remove extra parenthesis around `resolve`

File: apps/festival/festival-e2e/src/integration/dashboard/movie-tunnel.spec.ts (hunk 2 above)
Comment (xmano) [reply apps/festival/festival-e2e/src/integration/dashboard/movie-tunnel.spec.ts:259]: Fixed this.

File: apps/festival/festival-e2e/src/support/pages/dashboard/EventPage.ts (hunk 3 above)
Comment (xmano) [reply apps/festival/festival-e2e/src/support/pages/dashboard/EventPage.ts:11]: ok done.

File: apps/festival/festival-e2e/src/integration/dashboard/movie-tunnel.spec.ts (hunk 1 above)
Comment (xmano) [reply apps/festival/festival-e2e/src/integration/dashboard/movie-tunnel.spec.ts:232]: I need the function for debugging failures.. I will comment it out.

File: apps/festival/festival-e2e/src/integration/dashboard/movie-tunnel.spec.ts [hunk 4]
```diff
@@ -12,7 +12,7 @@ import { SEC } from '@blockframes/e2e/utils';
  */
 const debug_mode = false;
 
-const getStepsToSkip = (movie:any) => {
+const getStepsToSkip = (movie: Record<string, any>) => {
```
Comment (phpgeekfr) [inline apps/festival/festival-e2e/src/integration/dashboard/movie-tunnel.spec.ts:15]: you cannot use Movie or MovieDocument ?

**Review by phpgeekfr — APPROVED** [review]

**Pull request merged**